misc/python/materialize/scalability/endpoints.py

```diff
@@ -109,10 +115,24 @@ def priv_port(self) -> int:
 
     def up(self) -> None:
         self.composition.down(destroy_volumes=True)
+
+        try:
```

Comment:
Same as the other place -- try `git pull` first and make a decision based on that. `up` can fail for any number of other reasons.

Reply:
✅ Adjusted.